README.md

```diff
@@ -1,13 +1,11 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This was a project I did while attending the Technigo frontend bootcamp. The task was to start using a database to store and retrieve data from and use that data to produce a RESTful API.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I had the most problems creating error messages that would be sent to the user if the reguested data doesn´t excist. But with a little help from my team I solved it. 
```

Comment:
Readme "The Problem" is tricky. It is not about the problems we met during coding but about describing how we solve the given assignment and tools we used ;)
Read more here: https://www.notion.so/Writing-a-good-Readme-ef0b4488760f4ee6bc7842c31c3c0c66